scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -419,7 +419,7 @@ abstract class FixedPathSource(path: String*) extends FileSource {
   def localPaths = path.toList
   def hdfsPaths = path.toList
 
-  override def toString = getClass.getName + path
+  override def toString = getClass.getName + path.mkString(",")
```

Comment:
@rubanm I'll double check it - maybe if it only has one input source it fails
@Gabriel439 good call, I'll add that info. Not sure why WrappedArray is used though.